@vates/coalesce-calls/index.test.js

```diff
@@ -1,8 +1,10 @@
 'use strict'
 
-/* eslint-env jest */
+// eslint-disable-next-line n/no-unpublished-require
+const { describe, it } = require('test')
+const assert = require('assert')
 
-const { coalesceCalls } = require('./')
+const { coalesceCalls } = require('./index')
```

Comment:
Out of curiosity, why this change? :upside_down_face:

Reply:
No problem, it was just a question.

You did good by removing this unnecessary change :+1: 

My current practice is to explicit the full filename (e.g. `require("./index.js")`) because it will be necessary for ESM `import` (in `.mjs` files).